dist/docker/Dockerfile.archlinux

```diff
@@ -31,14 +30,11 @@ RUN pip3 install --upgrade pip && \
         install-python-deps
 
 # Install and Setup Conan
-ARG BUILD_TYPE=RelWithDebInfo
-RUN conan config init && \
-    conan profile update settings.build_type=${BUILD_TYPE} default && \
-    conan profile update settings.compiler.libcxx=libstdc++11 default && \
-    conan profile update settings.compiler.version=11 default && \
-    sed -r -i 's/10.1"]/10.1", "11", "11.1"]/' /root/.conan/settings.yml
-
+COPY dist/conan /cloe/dist/conan
+ARG CONAN_PROFILE=cloe-release
```

Comment:
- [ ] Replace `BUILD_TYPE` argument by `CONAN_PROFILE` in `dist/docker/Makefile`